src/btree/pagefile.ts

```diff
@@ -0,0 +1,44 @@
+import { RangeResolver } from "../resolver";
+
+const MAX_FREE_PAGE_INDICIES = 512;
+const PAGE_SIZE_BYTES = 4096;
+
+export class PageFile {
```

Comment:
feel like PageFile reader needs the most work. But I'm just generally stuck on how to approach this class